### PR TITLE
fix(api): Update pg to prevent silent failure on recent node versions during db initialization

### LINKED ISSUE #416: Issue with setting up local environment
### Describe the bug:

There are few issues with the local dev environment
- `await initDB()` fails silently without starting a server. If I replace the code with the snippet below, it works fine. 
```js
  initDB() 
  .then(() => {
    console.log("connected");
  }).catch((err) => {
    console.log(err);
  });
```
- `yarn both` gives me the following error 
```
$ yarn both
yarn run v1.22.5
$ concurrently "yarn dev" "yarn --cwd='./client' dev"
$ ts-node-dev --no-notify -P tsconfig.server.json ./server/app.ts
$ ts-node-dev --no-notify -P tsconfig.server.json ./server/app.ts
[0] ts-node-dev ver. 1.0.0-pre.63 (using ts-node ver. 8.10.2, typescript ver. 3.9.7)
[1] ts-node-dev ver. 1.0.0-pre.63 (using ts-node ver. 8.10.2, typescript ver. 3.9.7)
[1] in module
[0] in module
[1] After express
[0] After express
[0] DB initialized
[1] DB initialized
[0] built schema
[1] built schema
[0] Created server
[0] applied middleware
[0] prepared options.
[1] Created server
[1] applied middleware
[1] prepared options.
[0] Listening on http://localhost:5000/graphql
[1] Error: listen EADDRINUSE: address already in use :::5000
[1]     at Server.setupListenHandle [as _listen2] (net.js:1317:16)
[1]     at listenInCluster (net.js:1365:12)
[1]     at Server.listen (net.js:1451:7)
[1]     at Function.listen (D:\repos\chapter\node_modules\express\lib\application.js
:618:24)
[1]     at D:\repos\chapter\server\app.ts:62:9
[1]     at processTicksAndRejections (internal/process/task_queues.js:93:5)
[1] [ERROR] 19:53:59 Error: listen EADDRINUSE: address already in use :::5000

``` 
On the other hand, running both the servers in seperate terminals worked fine for me.
### Tell us about your browser and operating system:
- Browser(s) name and version: 
- Operating System: Windows 10 Home edition 
- Node Version: 14.13.10

### PR DESCRIPTION
**Pull request by devKate**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

Starting the server with node 14 or 15 results in an exit with no error. This is because `initDB()` calls typeorm's `createConnection` which is currently broken for older versions of `pg`, see
https://github.com/typeorm/typeorm/issues/5949 for more info. Updating `pg` resolves the issue but we may consider encouraging newcomers to rely on nvm/use a specific version of node to prevent these kinds of problems moving forward.

Related to one aspect of https://github.com/freeCodeCamp/chapter/issues/416

Edit By @vkWeb: 
Closes #416

### CODE REVIEW

**Comment by allella** [comment]
@devKate also asked on Discord

> are we locked into node versions above 13 or could we update nvmrc to point to 12.19.0? The docs are saying to install any node version >=13 but we might be able to help prevent this sort of issue by telling people to use nvm or the standalone lts version instead

**Comment by allella** [comment]
I've added a couple reviewers.  I can't add @Ryuno-Ki or Tom Noland but I'll message them in Discord

**Review by vkWeb — APPROVED** [review]
@devKate Thanks for fixing this long standing headache of ours! Me, Tomno Land, Rahul, @Ryuno-Ki, @timmyichen were all looking a solution for this.

@allella, @timmyichen I can confirm that this is a complete fix for #416. Server is communicating with database and client as expected. Dropping, migrations, seeds are working as expected too.

Screenshot from 2020-10-24 20-57-55
Screenshot from 2020-10-24 20-58-02

@devKate I would recommend moving forward with Node 14 as it will be in active LTS from Jan 2021. Our package.json says to use >=13.x which MUST be changed to >= 14. Node 13 is not in support anymore. Though we should see what @timmyichen has to say on this.

**Comment by allella** [comment]
@all-contributors please add @devKate for bug

**Comment by vkWeb** [comment]
@all-contributors please add @Ryuno-Ki for design and code.

File: .gitignore [hunk 1]
```diff
@@ -64,3 +64,6 @@ typings/
 .DS_Store
 
 .vscode/
+
+# Jetbrains project settings
+.idea/
```
Comment (timmyichen) [inline .gitignore:69]: nit: newline 🙃

**Review by timmyichen — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Ryuno-Ki) [reply .gitignore]: would it make sense to define an [EditorConfig](https://editorconfig.org/) file? (If so, I can open a new issue).

Comment (vkWeb) [reply .gitignore]: @Ryuno-Ki There's already one.

**Pull request merged**